store/src/main/java/org/apache/rocketmq/store/rocksdb/RocksDBOptionsFactory.java

```diff
@@ -71,7 +71,7 @@ public static ColumnFamilyOptions createCQCFOptions(final MessageStore messageSt
                 setTableFormatConfig(blockBasedTableConfig).
                 setMemTableConfig(new SkipListMemTableConfig()).
                 setCompressionType(CompressionType.LZ4_COMPRESSION).
-                setBottommostCompressionType(CompressionType.ZSTD_COMPRESSION).
+                setBottommostCompressionType(CompressionType.LZ4_COMPRESSION).
```

Comment:
这里为啥修改为LZ4，相对于iops，cpu更多是瓶颈吗？

Reply:
Does modifying the compression type impact in-place upgrade compatibility?

Reply:
Column Family for CQ uses Universal Compaction,  implying each SST will compact to last level. Unlike level compaction where ZSTD makes sense because it achieves better compression ratio at cost of more CPU usage.

LZ4 achieves more balanced CPU/compression ratio, thus is a better fit for  universal compaction.